input/resources/StructureDefinition-extension-qualification.json

```diff
@@ -0,0 +1,232 @@
+{
+  "resourceType": "StructureDefinition",
+  "id": "Qualification",
+  "url": "http://hl7.org/fhir/us/ndh/StructureDefinition/base-ext-qualification",
+  "name": "Qualification",
+  "title": "qualifications for an Organization",
+  "status": "draft",
+  "publisher": "HL7",
+  "description": "An extension to add qualifications for an organization (e.g. accreditation) or practitionerRole (e.g. registered to prescribe controlled substances).",
+  "fhirVersion": "4.0.1",
+  "mapping": [
+    {
+      "identity": "rim",
+      "uri": "http://hl7.org/comparison-v3.html",
+      "name": "RIM Mapping"
+    }
+  ],
+  "kind": "complex-type",
+  "abstract": false,
+  "context": [
+    {
+      "type": "element",
+      "expression": "Organization"
+    }
+  ],
+  "type": "Extension",
+  "baseDefinition": "http://hl7.org/fhir/StructureDefinition/Extension",
```

Comment:
change to "baseDefinition": "http://hl7.org/fhir/us/ndh/StructureDefinition/base-ext-qualification"

Reply:
Changed